lib/features/rules/BpmnRules.js

```diff
@@ -550,17 +550,18 @@ function canConnect(source, target, connection) {
     return connectDataAssociation;
   }
 
-  if (isCompensationBoundary(source) && isForCompensation(target)) {
-    return {
-      type: 'bpmn:Association',
-      associationDirection: 'One'
-    };
-  }
-
   if (canConnectAssociation(source, target)) {
 
+    if (isCompensationBoundary(source) && is(target, 'bpmn:Activity')) {
```

Comment:
I still don't understand why we cannot have association and compensate association as dedicated steps. 

As we treat both as distinct, why not clearly represent that in code?

Reply:
Makes sense. I will change that.